JesseMellinger/Crate
# Feature/user profile

#### What's this PR do?
- adds user menu button within the user view 
- add user account route 
- user button created and linked to user profile page 
- user account styling 

#### Where should the reviewer start?
- line 56 `Header.js` -- creates user header menu button to navigate to user profile page 
- line 1 `UserAccount.js` -- creates the `UserAccount` component / page 

#### How should this be manually tested?
- Check the user pages for all working links 
- Check styling of user account page compared to comp
- Make sure this component / page looks ready for connection to backend when ready

#### Any background context you want to provide?
- My prettier / beautify extension reformatted some of the files -- this might look like a lot of changes, but will probably have to go back and reformat -- I apologize for the inconvenience

#### Screenshots (if appropriate):
![Screen Shot 2021-02-08 at 11 26 04 AM](https://user-images.githubusercontent.com/67164959/107264655-775c1600-6a00-11eb-881a-af9e5b4a9d83.png)


#### Questions
